### PR TITLE
Added information links  (clock-app #19)

### PR DESCRIPTION
**Pull request by dark-eye**
Added the obligatory open source project links ( clock-app #19 )
- Added source link
- Added issues link
- Added translation link

### CODE REVIEW

File: app/components/Information.qml [hunk 1]
```diff
@@ -14,9 +14,9 @@ Page {
      }
 
     Component.onCompleted: {
-        infoModel.append({ name: i18n.tr("Get the source"), url: "https://github.com/ubports/clock-app" })
-        infoModel.append({ name: i18n.tr("Report Issues"), url: "https://github.com/ubports/clock-app/issues" })
-        infoModel.append({ name: i18n.tr("Help Translate"), url: "https://ubpweblate.tnvcomp.com/projects/ubports/clock-app/" })
+        infoModel.append({ name: i18n.tr("Get the source"), desc: i18n.tr("Source of clock lays here."), url: "https://github.com/ubports/clock-app" })
+        infoModel.append({ name: i18n.tr("Report issues"), desc: i18n.tr("Issues you have do you?"), url: "https://github.com/ubports/clock-app/issues" })
+        infoModel.append({ name: i18n.tr("Help translate"), desc: i18n.tr("Understand you I am not."), url: "https://translate.ubports.com/projects/ubports/clock-app/" })
     }
```
Comment (Flohack74) [inline app/components/Information.qml]: For the ironical desc: string, you would probably need to place a TRANSLATORS description what they should translate ;) - They will not have a clue and will ask me if this is really to be translated...
Like here: https://github.com/ubports/telegram-app/blob/master/telegram/po/com.ubuntu.telegram.pot#L37

**Comment by Flohack74** [comment]
Lets test now, if those new strings will appear automagically in Weblate ;)

**Comment by dark-eye** [comment]
@Flohack74 Hmm I think it doesn't work  still don't see the added strings, i think it`s because of this : https://docs.weblate.org/en/weblate-2.15/faq.html?highlight=template#why-does-weblate-still-shows-old-translation-strings-when-i-ve-updated-the-template

**Comment by Flohack74** [comment]
Argh! "Weblate also will not try to update bilingual files in any way, so if you need po files being updated from pot, you need to do it on your own."
So that means we cant do this automatically... hmmm.

**Comment by dark-eye** [comment]
I made a a script to update the po files from the pot (using msgmerge tool ) ill check if its working tonight...

**Pull request merged**